src/main/java/uc/doc/submitsolution/SubSDocDsRequestModel.java

```diff
@@ -2,24 +2,63 @@
 
 public class SubSDocDsRequestModel {
 
+    /**
+     * Name of the document
+     */
     private final String name;
 
+    /**
+     * The number of marks earned for this solution set
+     */
     private final Float recordedScore;
 
+    /**
+     * The ID of the user submitting this solution document
+     */
     private final String userId;
 
+    /**
+     * The ID of the course this document corresponds to
+     */
     private final String courseId;
 
+    /**
+     * The ID of the test that this solution document corresponds to
+     */
     private final String parentTestId;
 
+    /**
+     * The ID of the root message for the discussion board under this solution document
+     */
     private final String rootMessageId;
 
+    /**
+     * The total number of upvotes + downvotes for this solution document
+     */
     private final Integer voteTotal;
 
+    /**
+     * The path to the file on the submitting user's drive
+     */
     private final String filePath;
 
+    /**
+     * The estimated time it took to write this solution set
+     */
     private final Float estimatedTime;
 
+    /**
+     * Creates a new solution doc submission data storage model used to store the solution doc in a database
+     * @param name The name of the document
+     * @param userId The ID of the user submitting the document
+     * @param recordedScore The marks earned for this solution document
+     * @param courseId The ID of the course for which this solution document is in
+     * @param parentTestId The ID of the test which this solution document corresponds to
+     * @param filePath The path to the file on the submitting user's drive
+     * @param rootMessageId The ID of the root message for the discussion board attached to this solution document
+     * @param voteTotal The total number of upvotes and downvotes for this document
+     * @param estimatedTime The estimated time it took to write this solution set
+     */
```

Comment:
We weren't sure if the word database was too specific (for clean architecture) so we started using the term persistent data instead since we don't know if its specifically a database that is being used for storage? I'll double check with the tas on that on Monday though.

Reply:
Note: this is for line 51, I don't know what happened sorry.